src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -1103,7 +1107,11 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             return feedItems;
         }
         return feedItems.filter(item => {
-            return item.status === feedItemsStatusFilter || item.type === FEED_ITEM_TYPE_VERSION;
+            return (
+                item.status === feedItemsStatusFilter ||
+                item.type === FEED_ITEM_TYPE_VERSION ||
+                item.type === feedItemsStatusFilter
```

Comment:
Adding this last filter option adds basic support for filtering tasks. This doesn't re-fetch the activities each time a new filter is selected. Rather, it filters the current list of activities. New activities are only fetched on page reload.

Reply:
That makes sense, I could see that being beneficial in the long run. For the filtering, I think an important consideration is that within 1-2 sprints, we plan to implement a different strategy. I added this more as an MVP and bridge until we implement a proper filtering strategy.

I would lean towards adding a comment and leaving it as-is. Then, when updating the filtering strategy, change it to pass in that option (or something similar). I think it might be better to wait until we have more context into how the filtering will work and what would work best with the solution we implement.